### ISSUE TITLE
Make a v1.3.0 release of ics

### ISSUE THREAD

**Issue by faddat** [cosmos/interchain-security#933]
The ICS documentation clearly states the versioning practice as one where when breaking changes happen, a new minor release is cut, to ensure that there is not confusion among users. 

This is right and good. 

This release:

https://github.com/cosmos/interchain-security/releases/tag/v1.2.0-multiden

For whatever reason sits outside of those policies.  

The correct approach would be to take that, and the other changes that have accumulated in `main` and issue a new release.

**Comment by jtremback** [comment]
Agree

**Comment by smarshall-spitzbart** [comment]
Agreed @faddat, this issue will be superseded by https://github.com/cosmos/interchain-security/issues/949 which follows the EPIC template
